### PR TITLE
Windows compatibility for v3

### PR DESCRIPTION
**Pull request by mxstbr**

### CODE REVIEW

**Comment by mxstbr** [comment]
Can we have Travis CI run the tests on Windows too?

File: app/utils/tests/request.test.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ import request from '../request';
 import sinon from 'sinon';
 import expect from 'expect';
 
+import 'whatwg-fetch';
```
Comment (mxstbr) [inline app/utils/tests/request.test.js]: I'm pretty sure that isn't needed, we're stubbing fetch further down anyway

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+# Set build version format here instead of in the admin panel
+version: "{build}"
+
+# Which branches to build
+branches:
+
+  # Blacklist
+  except:
+    - master
```
Comment (mxstbr) [inline appveyor.yml]: We actually want to build master, I've disabled builds for branches that don't have an appveyor.yml so we can safely remove this I think.

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# Do not build on gh tags
+skip_tags: true
+ 
+# Test against these versions of Node.js
+environment:
+
+  matrix:
+    # Node versions to run
+    - nodejs_version: "5.0"
```
Comment (mxstbr) [inline appveyor.yml]: We should run on the same versions here as we run on travis

File: app/utils/tests/request.test.js (hunk 1 above)
Comment (jwinn) [inline app/utils/tests/request.test.js]: wanted to be consistent from a testing perspective

File: appveyor.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+# Disable automatic builds
+build: off
+
+# Post-install test scripts
+test_script:
+  # Output debugging info
+  - node --version
+  - npm --version
+  # run tests
+  - npm test
```
Comment (mxstbr) [inline appveyor.yml]: We should run `npm run build` here, which runs `npm test` anyway

File: app/utils/tests/request.test.js (hunk 1 above)
Comment (mxstbr) [inline app/utils/tests/request.test.js]: Still, lets remove it so we don't confuse anybody.

**Comment by mxstbr** [comment]
Awesome, awesome work, thanks for fighting through this, very appreciated!

**Comment by w01fgang** [comment]
Now on Mac `npm run setup` fails with an error `sh: cross-env: command not found`
I think we need to add `cross-env` in the prerequisites section of the documentation or in installation instructions.

**Comment by mxstbr** [comment]
Oh good catch, is it not in the dependencies?

**Comment by mxstbr** [comment]
It is, so that *should* work...

**Comment by mxstbr** [comment]
Damn I can replicate, it doesn't work. Give me a second.

**Comment by mxstbr** [comment]
Right, the problem is that the `npm run setup` script uses cross-env, which isn't installed at that point because `npm run setup` installs the dependencies

**Comment by w01fgang** [comment]
yes, we must to check first if `cross-env` is installed

**Comment by mxstbr** [comment]
Fixed in 2d4c95d

**Comment by mxstbr** [comment]
Thanks for notifying me of this, good catch!

**Comment by w01fgang** [comment]
glad to help :+1:

**Pull request merged**